src/data_morph/shapes/points.py

```diff
@@ -304,3 +305,67 @@ def distance(self, x: Number, y: Number) -> int:
             Always returns 0 to allow for scattering of the points.
         """
         return 0
+
+
+class Spade(PointCollection):
+    """
+    Class for the spade shape.
+
+    .. plot::
+       :scale: 75
+       :caption:
+            This shape is generated using the panda dataset.
+
+        from data_morph.data.loader import DataLoader
+        from data_morph.shapes.points import Heart
+
+        _ = Spade(DataLoader.load_dataset('panda')).plot()
+
+    Parameters
+    ----------
+    dataset : Dataset
+        The starting dataset to morph into other shapes.
+    """
+
+    def __init__(self, dataset: Dataset) -> None:
+        x_bounds = dataset.data_bounds.x_bounds
+        y_bounds = dataset.data_bounds.y_bounds
+
+        # Graph heart curve
+        x_shift = sum(x_bounds) / 2
+        y_shift = sum(y_bounds) / 2
+
+        t = np.linspace(-3, 3, num=90)
+
+        heart_x = -(16 * np.sin(t) ** 3)
+        heart_y = -(
+            13 * np.cos(t) - 5 * np.cos(2 * t) - 2 * np.cos(3 * t) - np.cos(4 * t)
+        )
+
+        # Graph line base
+        line_t = np.linspace(-6, 6, num=12)
+        line_x = line_t
+        line_y = [-16 for _ in line_t]
+
+        # Graph left wing
+        left_t = np.linspace(-6, 0, num=12)
+        left_x = left_t
+        left_y = [0.278 * math.pow(point + 6.0, 2) - 16.0 for point in left_t]
```

Comment:
Let's use `numpy` instead of `math` and vectorize this. https://numpy.org/doc/stable/reference/generated/numpy.power.html